recipe/meta.yaml

```diff
@@ -14,19 +14,17 @@ source:
     - sdl2path.patch
 
 build:
-  number: 1002
-  script: python setup.py install --record record.txt
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   build:
     - patch  # [win]
-    - sdl2
+  host:
+    - pip
     - python
-    - setuptools
-
   run:
     - sdl2
-    - python
```

Comment:
This is your problem.

Reply:
So python should be added to run, even though it's also present under `host`. Got it. To be honest I'm a bit confused with what goes where at the moment. Thanks for your help!

Reply:
`host` what must be on the host machine during build. If you depend on python to run you must add it to `run`. For more information see https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run